frontend/src/app/main/data/workspace/annotation_helpers.cljs

```diff
@@ -0,0 +1,46 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) KALEIDOS INC
+
+(ns app.main.data.workspace.annotation-helpers
+  (:require
+   [app.common.data.macros :as dm]
+   [app.common.logging :as log]
+   [app.common.types.components-list :as ctkl]
+   [app.main.refs :as refs]))
+
+;; Change this to :info :debug or :trace to debug this module, or :warn to reset to default
+(log/set-level! :warn)
+
+(defn- get-libraries
```

Comment:
Perhaps all this may be simplified a good bit. We can talk about it.

Reply:
Done